features/uri-templating.feature

```diff
@@ -2,68 +2,79 @@
 Functionaliteit: URI templating
     Als API provider
     wil ik uri templating kunnen gebruiken
-    zodat gelijksoortige links kunnen worden gerepresenteerd door één URI template
+    zodat grote aantallen links naar dezelfde API in de response kunnen worden gerepresenteerd door één URI template
+    zodat ik geen url's naar externe API's hoeft te beheren
 
     Zoals gespecificeerd in RFC 6570 (https://tools.ietf.org/html/rfc6570):
     - Een URI template is een string die 0 of meer expressies bevat, waarmee een verzameling van URI's kan worden beschreven
     - Een expression is de tekst tussen een open accolade '{' en een sluit accolade '}' inclusief de accolade tekens
-    - Expanden van een template is het vervangen van een expression door een waarde. De manier van expanden wordt bepaald door een optionele operator in de expression. Voor meer info zie: https://tools.ietf.org/html/rfc6570#section-2.2
+    - Expanden van een template is het vervangen van een expression door een waarde.
+      De manier van expanden wordt bepaald door een optionele operator in de expression. Voor meer info zie: [Expressions](https://tools.ietf.org/html/rfc6570#section-2.2)
 
     Op het moment van specificeren (18-03-2020) wordt in de Haal Centraal API's alleen de meest simpele 'Simple String Expansion' expression gebruikt,
     een {expression} wordt vervangen door een variabele
 
-    De OpenAPI Specifications Server Object ondersteunt ook URI templating (https://swagger.io/specification/#serverObject).
-    In de url property kunnen één of meerdere expressions worden gespecificeerd. De mogelijke waarden voor elke expression worden in de variables map gespecificeerd.
-    In onderstaande scenario's wordt voor de Server objecten URI templating toegepast.
+    Afspraken:
+    - gebruik {[xxx]serverurl} als placeholder voor de server url van externe API url's
+      [xxx] is de afkorting die wordt gebruikt voor de externe API, bijv. brp, bag, brk
```

Comment:
is het niet vriendelijker de exacte lijst te benoemen? Nu staat er bijvoorbeeld, dus zou een client deze, maar mogelijk ook andere moeten ondersteunen. Bovendien wordt daarmee expliciet dat het uitbreiden van deze lijst in sommige gevallen een breaking change is.
Ik denk dat de lijst dan is:
- {brpserverurl}
- {bagserverurl}
- {brkserverurl}
- {hrserverurl}

Reply:
check